components/experimental/src/transliterate/transliterator/mod.rs

```diff
@@ -341,7 +351,7 @@ impl Transliterator {
     fn load_special<P>(
         special: &str,
         normalizer_provider: &P,
-    ) -> Result<InternalTransliterator, DataError>
+    ) -> Result<Option<InternalTransliterator>, DataError>
```

Comment:
I think this should be `Option<Result<InternalTransliterator, DataError>>`. The option signals that a special transliterator is implemented, and then the result is the result from instantiating that transliterator, i.e. the `_` case should just be `None`.